pkgs/servers/ankisyncd/default.nix

```diff
@@ -46,14 +47,14 @@ python3.pkgs.buildPythonApplication rec {
 
   buildInputs = [ ];
 
-  propagatedBuildInputs = [ anki ];
+  propagatedBuildInputs = with python3.pkgs; [
+    decorator
+    requests
+  ];
 
   checkPhase = ''
-    # Exclude tests that require sqlite's sqldiff command, since
-    # it isn't yet packaged for NixOS, although 2 PRs exist:
-    # - https://github.com/NixOS/nixpkgs/pull/69112
-    # - https://github.com/NixOS/nixpkgs/pull/75784
-    # Once this is merged, these tests can be run as well.
+    # skip these tests, our files are too young:
+    # tests/test_web_media.py::SyncAppFunctionalMediaTest::test_sync_mediaChanges ValueError: ZIP does not support timestamps before 1980
```

Comment:
See https://nixos.org/manual/nixpkgs/unstable/#python-setup.py-bdist_wheel-cannot-create-.whl

You can set `SOURCE_DATE_EPOCH = 315532800;` to workaround.

Reply:
Just setting that wasn't enough, the test still has the same error.

I'm pretty inclined to not spend any more time on this since I'm hoping to delete that package pretty soon anyway.